src/components/Col/Col.test.tsx

```diff
@@ -1,72 +1,76 @@
-import { shallow } from "enzyme";
+import { render, screen } from "@testing-library/react";
 import React from "react";
 
 import Col from "./Col";
 
 describe("Col ", () => {
   it("renders", () => {
-    const wrapper = shallow(<Col size={3}>Test content</Col>);
-    expect(wrapper).toMatchSnapshot();
+    render(
+      <Col data-testid="col" size={3}>
+        Test content
+      </Col>
+    );
+    expect(screen.getByTestId("col")).toMatchSnapshot();
```

Comment:
This is a good way to do this I think. Set test id in the test itself, and use it to get an element.